scalableminds/webknossos
# Mark owner in user list

PR adds a simple change to the user list. "Organization Owner" is now a an additional tag.

<img width="1612" alt="image" src="https://user-images.githubusercontent.com/1105056/217514519-12f3e98b-9d22-4f9f-aeca-f913c7760855.png">


### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Switch to user list 
- Check layouting
- Edit the roles / teams of a test user

### Issues:
- contributes to #6509 

------
(Please delete unneeded items, merge only when none are left open)
